docs/core/introduction.md

```diff
@@ -179,231 +77,172 @@ A .NET app is built from source code by using [MSBuild](/visualstudio/msbuild/ms
+* [The Microsoft download site](https://dotnet.microsoft.com/download/)
+* [Containers](https://hub.docker.com/_/microsoft-dotnet)
+* [Linux package managers](install/linux.md).
 
-Higher-level .NET languages, such as C#, compile down to a hardware-agnostic instruction set, which is called Intermediate Language (IL). When an app runs, the JIT compiler translates IL to machine code that the processor understands. JIT compilation happens on the same machine that the code is going to run on.
+## Support
 
-Since JIT compilation occurs during execution of the application, the compilation time is part of the run time. Therefore, JIT compilers have to balance time spent optimizing code against the savings that the resulting code can produce. But a JIT compiler knows the actual hardware and can free developers from having to ship different implementations for different platforms.
+[Microsoft supports .NET](https://github.com/dotnet/core/blob/main/os-lifecycle-policy.md) on Android, Apple, Linux, and Windows operating systems. It can be used on Arm64, x64, and x86 chips. It is also supported in emulated environments, like [macOS Rosetta 2](https://support.apple.com/HT211861).
 
-The .NET JIT compiler can do *tiered compilation*, which means it can recompile individual methods at run time. This feature lets it compile quickly while still being able to produce a highly tuned version of the code for frequently used methods.
+New versions of .NET are released annually in November. .NET releases in odd-numbered years are Long-term Support (LTS) releases and are supported for three years. Releases in even-numbered years are Short-team Support (STS) releases and are supported for 18 months. All other aspects of the releases are the same, like quality level and breaking change policies. For more information, see [Releases and support](releases-and-support.md).
 
-For more information, see [Managed execution process](../standard/managed-execution-process.md) and [Tiered compilation](whats-new/dotnet-core-3-0.md#tiered-compilation).
+The .NET Team at Microsoft works collaboratively with other organizations to distribute and support .NET in various ways.
 
-### AOT compiler
+[Red Hat supports .NET](https://developers.redhat.com/topics/dotnet/) on Red Hat Enterprise Linux (RHEL).
 
-The default experience for most .NET workloads is the JIT compiler, but .NET offers two forms of ahead-of-time (AOT) compilation:
+[Samsung supports .NET](https://developer.tizen.org/development/training/.net-application) on Tizen platforms.
 
-* Some scenarios require 100% AOT compilation. An example is [iOS](/xamarin/ios/).
-* In other scenarios, most of an app's code is AOT-compiled but some is JIT-compiled. Some code patterns aren't friendly to AOT (like generics). An example of this form of AOT compilation is the [ready-to-run](whats-new/dotnet-core-3-0.md#readytorun-images) publish option. This form of AOT offers the benefits of AOT without its drawbacks.
+## Runtime technology
```

Comment:
Not sure if it is critical for this intro doc, but there are runtime technology(s) in our stack: coreclr, mono, native aot, etc.  Perhaps a note that the runtime used is tailored to the scenario.

Reply:
There was more of that in the current doc, but it is super confusing. I think a new doc with that info is warranted.

https://docs.microsoft.com/en-us/dotnet/core/introduction#net-core-net-framework-mono-uwp

Reply:
I added a (starter) section at the end.